### PR TITLE
Session type test

### PR DESCRIPTION
**Pull request by agagancarczyk**
## Motivation
Add cypress tests for session type dropdown displaying on the sessions page.

## Brief Description
Adding cypress tests for session type dropdown.

## Verification Steps
Run sessions_test.spec.ts. All tests should be green as per attached

Screenshot 2021-07-14 at 11 13 05

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated
- [ ] Formatting has been performed via prettier/eslint
- [ ] Type checking has been performed via 'npm run check-types'

### CODE REVIEW

File: cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  regularSSOOption: string;
+  offlineOption: string;
+  directGrantOption: string;
+  serviceAccountOption: string;
+  selectedType: string;
+
+  constructor() {
+    this.sessionTypeDrpDwn = ".pf-c-select__toggle";
+    this.sessionTypeList = ".pf-c-select__toggle + ul";
+    this.allSessionTypesOption = '[data-testid="all-sessions-option"]';
```
Comment (edewit) [inline cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts:14]:
```suggestion
    this.allSessionTypesOption = "all-sessions-option";
```

File: cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  shouldNotBeEmpty() {
+    cy.get(this.sessionTypeDrpDwn).click();
+    cy.get(this.sessionTypeList).should("exist");
+
+    return this;
+  }
+
+  selectAllSessionsType() {
+    cy.get(this.sessionTypeDrpDwn).click();
+    cy.get(this.allSessionTypesOption).click();
```
Comment (edewit) [inline cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts:35]:
```suggestion
    cy.getId(this.allSessionTypesOption).click();
```

**Review by edewit — CHANGES_REQUESTED** [review]
when using `data-testid` you can use `cy.getId` this is a helper function in `./cypress/support/commands.js` that prefixes the `data-testid`

File: cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts (hunk 1 above)
Comment (agagancarczyk) [reply cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts]: great catch! I'll make this change now.

File: cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts (hunk 2 above)
Comment (agagancarczyk) [reply cypress/support/pages/admin_console/manage/sessions/SessionsPage.ts]: this is changed now for all options.

**Review by agagancarczyk — COMMENTED** [review]
this is good to know!

**Review by edewit — APPROVED** [review]

**Pull request merged**